contracts/monitoring/src/error.rs

```diff
@@ -0,0 +1,19 @@
+use axelar_wasm_std::nonempty;
+use axelar_wasm_std_derive::IntoContractError;
+use cosmwasm_std::{OverflowError, StdError};
+use thiserror::Error;
+
+#[derive(Error, Debug, PartialEq, IntoContractError)]
+pub enum ContractError {
+    #[error(transparent)]
+    Std(#[from] StdError),
+
+    #[error(transparent)]
+    Overflow(#[from] OverflowError),
+
+    #[error(transparent)]
+    NonEmpty(#[from] nonempty::Error),
```

Comment:
non of these are used, don't add them preemptively

Reply:
```
#[error(transparent)]
Std(#[from] StdError),
```
is used Chris, in checking for governance address, when loading from config:
`let config = CONFIG.load(deps.storage)?;`
if I remove `Std` error, the load fails. But I have removed the two other ones you mentioned.